test/test-applications/integrations/aspnet/TestApplication.AspNet/Web.config

```diff
@@ -68,6 +68,10 @@
         <assemblyIdentity name="System.Web.Mvc" publicKeyToken="31bf3856ad364e35" />
         <bindingRedirect oldVersion="1.0.0.0-5.2.7.0" newVersion="5.2.7.0" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="System.Numerics.Vectors" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
```

Comment:
Is this a new dependency added to 1.2.0?

Reply:
If it's a new dependency, it's a very indirect one. It was added when I triggered the update for the OTel dependencies via the Nuget package manager UI. Unfortunately I could not run that test locally to confirm whether or not it was really necessary.

Reply:
@pjanotti discovered that there was a build warning on our main branch related to the "System.Numerics.Vectors" binding redirect. The Nuget package management tool in the Visual Studio UI resolved that warning for us automatically by adding this binding redirect. It is not strictly necessary for the purposes of this PR, but it is still useful.